hack/mirror-chart.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# Copyright 2025 The Kubermatic Kubernetes Platform contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This helper script can be used to mirror upstream system application Helm charts to Kubermatic OCI registry.
+# When introducing a new helm chart version for our system applications, make sure it is:
+#   - Helm chart is mirrored in Kubermatic OCI registry, use the script hack/mirror-chart.sh <chart-name>
```

Comment:
I'm not sure if you should write it like this within the script itself. Just clearly document what needs to change for adding a new chart and mirror a new version of chart.